changelog/fragments/3173-bugfix.yaml

```diff
@@ -0,0 +1,17 @@
+entries:
+  - description: >
+      Prevent the ansible-operator from mangling variables containing the uppercase representations of special words (e.g IP, HTTP, etc).
+    kind: change
+    breaking: true
```

Comment:
Is this really breaking? Does an operator author have to change anything in an existing project?

Reply:
Yeah, it could be changing the parameters that are passed in, which means they could be referencing a variable that no longer exists. The only workaround for this bug was to reference the improperly converted variable so I expect anyone that hit it and continued going will be affected.